package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "zoapp-backend",
-  "version": "0.11.5",
+  "version": "0.11.6",
```

Comment:
https://github.com/Zoapp/react-materialcomponents#how-to-create-development-releases 😋

Reply:
It was the previous tests I made with npm version patch. I didn't change the version since, but I didn't commited also. 😋